### ISSUE TITLE
ordinal@^1.0.0 - no compatible version found

### ISSUE THREAD

**Issue by dottorblaster** [apparatus/fuge#67]
Hi,
Is this me, or package.json needs to be fixed in order to include the right ordinal version?

```
$ npm install
npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/Cellar/node/5.7.0/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v5.7.0
npm ERR! npm  v3.6.0

npm ERR! No compatible version found: ordinal@^1.0.0
npm ERR! Valid install targets:
npm ERR! 0.0.4, 0.0.3, 0.0.2, 0.0.1, 0.0.0
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/blaster/Desktop/projects/fuge/npm-debug.log
```

**Comment by dottorblaster** [comment]
```
$ npm --version
3.8.3
```

**Comment by dottorblaster** [comment]
Fixed in #68, got merged.
